style.css

```diff
@@ -1,3 +1,103 @@
-#root {
-  color: red;
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;600&display=swap');
+
+/* font-family: 'Rubik', sans-serif; */
+
+:root{
+--Royalblue: #3f3f3e;
+--white: #e3e3e3;
+--darkGrey: #1c1c1c;
+--fyellow: #fcad00
+}
+/* Main  */
+body{
+font-family: 'Rubik', sans-serif;
+background-color: var(--darkGrey);
+color: antiquewhite;
+margin: 0;
+padding: 0;
+}
+a {
+    color: var(--fyellow);
+  }
+/* Header */
+header{
+    display: flex;
+    align-content: space-around;
+    flex-wrap: wrap;
+    align-items: center;
+    flex-direction: column;
+}
+.logo{
+    font-size: 50px;
+    margin: 0;
+}
+#logo-one-split{
+    color: var(--fyellow);
+}
+#logo-two-split{
+    color: var(--fyellow);
+}
+/* Movie Card */
+#container{
+    display: flex; 
+}
+ul{
+    display: flex;
+    flex-wrap: wrap;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    
+}
+li{
+    flex: auto ;
+    margin: 10px;
+    width: 280px;
+    padding: 20px;
+    background: var(--Royalblue);
+    border-radius: 5px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.05);
+}
+.epi-format{
+    color: var(--fyellow);
+    font-size: 15px;
+}
+.summary{
+ font-size: 15px; 
+}
+.episode-name{
+    border-bottom: var(--fyellow) solid 2px;
+    font-size: 18px; 
+}
+.backButton{
+    margin-top:10px ;
+    padding: 5px 5px 5px 5px; 
+    border: none;
+    font-size: 15px;
+    font-family: 'Rubik', sans-serif;
+    font-weight: 300;
+    background-color: var(--fyellow);
+    color: var(--Royalblue);  
+    border-radius: 5px;
+   }
+   #selector{
+   padding: 5px 5px 5px 5px;  
+   font-family: 'Rubik', sans-serif;
+   background-color: var(--fyellow);
+   border-radius: 15px;
+   color: var(--Royalblue); 
+   }
+/* Footer */
+footer{
+    background-color: var(--fyellow);
+    color: var(--darkGrey);
+    display: flex;
+    justify-content: center;
+}
+#searchBar{
+padding: 10px;
+font-family: 'Rubik', sans-serif;
+font-size: 20px;
+font-weight: 500;
+color: var(--Royalblue);
```

Comment:
Hi Milad, you did an amazing job. I just checked your web and it's awesome. :) One thing I would like to suggest is title border line. Instead of bottom line, It would be nicer with border box in my opinion. It's not about JS and it's a small tiny thing in CSS.